#!/usr/bin/env python
PACKAGE = "rmal"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("x_max", double_t, 0, "x_max",    0.3, 0,   2)
gen.add("x_min", double_t, 0, "x_min",    -0.44, -2,  0)

gen.add("y_max", double_t, 0, "y_max",    0.24, 0,   2)
gen.add("y_min", double_t, 0, "y_min",    -1.0, -2,  0)

gen.add("z_max", double_t, 0, "z_max",   1.08, 0,   2)
gen.add("z_min", double_t, 0, "z_min",    -1.0, -2,  0)

exit(gen.generate(PACKAGE, "cropping_point_cloud", "croppingCloud"))
